src/modules/new-request-form/ticket-form-field/TicketFormField.tsx

```diff
@@ -6,38 +6,27 @@ import {
   Label,
   Option,
 } from "@zendeskgarden/react-dropdowns.next";
-import type { TicketForm } from "../data-types/TicketForm";
 import type { Field } from "../data-types";
 
 interface TicketFormFieldProps {
-  label: string;
-  ticketFormField: Field;
-  ticketForms: TicketForm[];
+  field: Field;
 }
 
 const key = "return-focus-to-ticket-form-field";
 
-export function TicketFormField({
-  label,
-  ticketFormField,
-  ticketForms,
-}: TicketFormFieldProps) {
+export function TicketFormField({ field }: TicketFormFieldProps) {
   const ref = createRef<HTMLDivElement>();
 
   const handleChange: IComboboxProps["onChange"] = ({ selectionValue }) => {
     if (selectionValue && typeof selectionValue === "string") {
-      const newUrl = new URL(window.location.origin + selectionValue);
+      const url = new URL(window.location.href);
```

Comment:
Good catch! This is due to me changing it at the last minute from `string` to `number` in hc.
`number` is what we get from the api but we then mostly handle `string` in the app. Are you suggesting we should expose `string` instead of `number`?

Reply:
Maybe it would be better to always return strings since the type is defined as a string [here](https://github.com/zendesk/copenhagen_theme/blob/1fc4c2b0a9c94feb91143da3157b179908ad22ca/src/modules/new-request-form/data-types/Field.ts#L15). I am ok in addressing it in another PR.